src/controllers/translateController.ts

```diff
@@ -10,8 +10,9 @@ import {
 import { validateLanguageRequested } from "../utils/supportedLanguagesUtils";
 import { validEngines } from "../utils/translationEnginesUtils";
 import { TranslateController } from "./translateController.d";
-import { Settings } from "../types/settings";
+import { Settings, SortOrder } from "../types/settings";
```

Comment:
you can make reference to the .d file directly, so you avoid the change on jest config

```suggestion
import { Settings, SortOrder } from "../types/settings.d";
```